pmm/aws-staging-start.groovy

```diff
@@ -289,6 +289,7 @@ pipeline {
                                             ${DOCKER_VERSION}
                                         sleep 10
                                         docker logs \${VM_NAME}-server
+                                        docker exec \${VM_NAME}-server change-admin-password \${ADMIN_PASSWORD}
```

Comment:
change-admin-password utility is only available in the latest PMM version, but if I want to launch an older PMM-Version it won't work.

Reply:
fixed thanks